### ISSUE TITLE
Problems connecting Tesla WiFi to Tesla-Android

### ISSUE THREAD

**Issue by Dazzert318** [tesla-android/issue-tracker#36]
I’ve successfully built Tesla-Android and been able to connect to it from my laptop and iPad using chrome and safari. However, when I deploy it to my vehicle I cannot connect the Tesla WiFi to Tesla-Android. It keeps reporting either ‘could not connect  to network’ or ‘unable to obtain an IP address’. Any advice please as to what’s going wrong?

**Comment by mbhforum** [comment]
I had the same issue.  Make sure your LTE modem is connecting to your carrier.  Take the USB stick and plug into your computer to see the Connection Status. In my case, I had to call AT&T and have them activate it properly and once I did that, it resolved the issue.  Tesla will apparently not connect to a Wifi network that doesn't have a valid Internet connection.  My guess is this is a security feature so Tesla doesn't connect to a compromised hotspot, etc.
